src/format.ts

```diff
@@ -184,8 +183,8 @@ export class ModelFormat<T extends Entity> extends Format<T> {
 		this.type = type;
 		this.formatEval = formatEval;
 
-		// Compile the model format
-		this.compile();
+		// Compile the model format when the model is ready
```

Comment:
Only compile the format when the model is ready to ensure properties referenced by tokens have been created/initialized. `enqueueWhileProcessing: false` allows the format to be compiled immediately if it is created after the model is ready, but the callback queue is currently being processed.